apps/src/music/views/TimelineSimple2Events.jsx

```diff
@@ -50,8 +50,9 @@ const TimelineSimple2Events = ({
   for (const soundEvent of soundEvents) {
     const soundId = soundEvent.id;
     const functionName = soundEvent.functionContext.name;
+    // TODO: Add length as field on PlaybackEvent to prevent duplicated lookup logic
     const length =
-      soundEvent.type === 'pattern'
+      soundEvent.type === 'pattern' || soundEvent.type === 'chord'
         ? DEFAULT_PATTERN_LENGTH
```

Comment:
Are we okay not explicitly using `DEFAULT_CHORD_LENGTH` here?

Reply:
Oh good catch, I had this initially just to get things working before adding DEFAULT_CHORD_LENGTH. I actually may just leave this as is since I plan to fast-follow with the length caching fix.